src/compas_view2/app/controller.py

```diff
@@ -115,131 +235,114 @@ def glsl_version(self):
         self.app.info(value)
 
     # ==============================================================================
-    # View actions
+    # Key mouse actions
     # ==============================================================================
 
-    def view_shaded(self):
-        """Switch the view to shaded.
-
-        Returns
-        -------
-        None
-
-        """
-        self.app.view.mode = "shaded"
-        self.app.view.update()
-
-    def view_ghosted(self):
-        """Switch the view to ghosted.
-
-        Returns
-        -------
-        None
-
-        """
-        self.app.view.mode = "ghosted"
-        self.app.view.update()
-
-    def view_wireframe(self):
-        """Switch the view to wireframe.
-
-        Returns
-        -------
-        None
-
-        """
-        self.app.view.mode = "wireframe"
-        self.app.view.update()
-
-    def view_lighted(self):
-        """Switch the view to lighted.
-
-        Returns
-        -------
-        None
-
-        """
-        self.app.view.mode = "lighted"
```

Comment:
Why are these ones deleted here?

Reply:
yes! you are right actually. somehow i mixed the code at the last moment when i compareed the files. This commit should fix them!